### ISSUE TITLE
Serial port not detected

### ISSUE THREAD

**Issue by nardusleroux** [ParadoxAlarmInterface/hassio-repository#6]
Hi,
Thanks for great work to add PAI to hass.io!

Issue solved, but thought that I will add instructions on how to overcome this. Not sure whether this will make sense to add in future release since highly dependent on setup.

I am using the Paradox 306USB converter which exposes itself as ttyUSB0 on host and addon complained about port not being available. 
I added the following line to config.json
`"devices": ["/dev/ttyUSB0:/dev/ttyUSB0:rwm"],`

Also had to add udev rule on host to allow non root user access to serial device. I used instructions as per [https://stackoverflow.com/questions/24225647/docker-a-way-to-give-access-to-a-host-usb-or-serial-device](url)

**Comment by yozik04** [comment]
Probably for hass.io OS you would need to use privileged mode, because you do not have access to udev rules.

**Comment by yozik04** [comment]
It is not possible to add `devices` to the `config.json` as it would require the device to exist on the host to start the container.
Anyway linked to this issue from the wiki.

**Comment by nardusleroux** [comment]
Thanks, should have added, I am running Hasbian with docker and hassio on top of that, not hassos.

**Comment by cybertza** [comment]
Hey, so this is how i went about a similar install:
https://github.com/ParadoxAlarmInterface/pai/wiki/Connection-methods#serial-over-ip-esp32

but i guess you could make a different TCP: serial on the host with socat or alike and then address it as a IP_CONNECTION_BARE = True

**Comment by cybertza** [comment]
https://www.losant.com/blog/how-to-access-serial-devices-in-docker
let me know if this works?

**Comment by cybertza** [comment]
i see the IDE addon has this setting available, 
![image](https://user-images.githubusercontent.com/20600538/72687795-3e59b680-3b0a-11ea-93ec-b764c0716abb.png)

so i guess it may be possible to add for direct serial access?

**Comment by cybertza** [comment]
Ok so, here seems to be the solution, albeit i think you may have try this and let us know:
https://developers.home-assistant.io/docs/en/hassio_addon_config.html

```
devices | list | no | Device list to map into the add-on. Format is: <path_on_host>:<path_in_container>:<cgroup_permissions>. i.e. /dev/ttyAMA0:/dev/ttyAMA0:rwm
-- | -- | -- | --
```
![image](https://user-images.githubusercontent.com/20600538/72687931-6564b800-3b0b-11ea-8847-a99add825ea7.png)

**Comment by cybertza** [comment]
Ok so, here seems to be the solution, albeit i think you may have try this and let us know:
https://developers.home-assistant.io/docs/en/hassio_addon_config.html

```
devices | list | no | Device list to map into the add-on. Format is: <path_on_host>:<path_in_container>:<cgroup_permissions>. i.e. /dev/ttyAMA0:/dev/ttyAMA0:rwm
-- | -- | -- | --
```

**Comment by yozik04** [comment]
See my post https://github.com/ParadoxAlarmInterface/hassio-repository/issues/6#issuecomment-573293583

**Comment by yozik04** [comment]
I fixed it proper way.

**Comment by yozik04** [comment]
Released in 2.0.0
